examples/cookbook/cookbook/urls.py

```diff
@@ -1,9 +1,10 @@
-from django.conf.urls import url
 from django.contrib import admin
+from django.urls import re_path
+from django.views.decorators.csrf import csrf_exempt
 
-from graphene_django.views import GraphQLView
+from graphene_django.views import AsyncGraphQLView
 
 urlpatterns = [
-    url(r"^admin/", admin.site.urls),
-    url(r"^graphql$", GraphQLView.as_view(graphiql=True)),
+    re_path(r"^admin/", admin.site.urls),
+    re_path(r"^graphql$", csrf_exempt(AsyncGraphQLView.as_view(graphiql=True))),
```

Comment:
Do we want to make the async view the only, and default, one in the cookbook? I feel we should have both examples present, the sync one will most likely still be the most common one.

Reply:
@jaw9c I can help add an example with the sync view. I think having both examples present makes sense.

Reply:
@jaw9c I created a PR to have both an async & sync example of the cookbook. I basically took these changes, stuck them in `cookbook-async` and put the sync `GraphQLView` back. Should be good to merge into this PR, but let me know what you think!